test/test_hpack.py

```diff
@@ -36,14 +36,22 @@ def test_sensitive_headers(self):
         Test encoding header values  
         """
         e = Encoder()
-        result = b'\x82\x14\x88\x63\xa1\xa9\x32\x08\x73\xd0\xc7\x10\x87\x25\xa8\x49\xe9\xea\x5f\x5f\x89\x41\x6a\x41\x92\x6e\xe5\x35\x52\x9f'
+        result = (b'\x82\x14\x88\x63\xa1\xa9' +
+                  b'\x32\x08\x73\xd0\xc7\x10' +
+                  b'\x87\x25\xa8\x49\xe9\xea' +
+                  b'\x5f\x5f\x89\x41\x6a\x41' +
+                  b'\x92\x6e\xe5\x35\x52\x9f')
         header_set = [
             (':method', 'GET', True),
             (':path', '/jimiscool/', True),
             ('customkey','sensitiveinfo',True) 
         ]
         assert e.encode(header_set, huffman=True) == result
-        
+
+    def test_header_table_size_getter(self):
+        e = Encoder()
+        htsize = e.header_table_size
+        assert htsize == e.header_table.maxsize
```

Comment:
Rather than assert that these two are equal, let's assert the return value. We don't really care where it came from. =) (We care for coverage, of course, but we should assert observables wherever possible, not implementation.)